doc/alert_disk_space_template.jinja2

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# alert if / has less than 7.2G available
+
+#available=$(df /home|grep /home|awk '{ print $4 }')
+available=$(df /dev/root|grep /dev/root|awk '{ print $4 }')
+if [ $available -le 7460500 ]; then
```

Comment:
`7460500` should become a template value, set from rc files. You can use GB in templates, and have fabric calculate this value in kb.